src/main/resources/static/style.css

```diff
@@ -0,0 +1,231 @@
+button {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 90%;
+    background-color: rgb(250, 250, 255);
+    padding: 1.9%;
+    margin: 0.5%;
+    border-color: aliceblue;
+    box-shadow: none;
+    color: rgb(95, 95, 97);
+    border-radius: 0.2em;
+}
+
+#confirmChanges {
```

Comment:
Generally it's better practice to use .class selector rather than #id selectors, but I don't have strong reason why, so we can keep it :D
I guess it's because it's more likely to give 2 different elements same class (and reuse some styles), than to give 2 different elements same id

Reply:
Oh, that's why - I thought I was missing something - that way alle the pads (Account, Equation etc. can have the same class)